tools/workspace-tests/verify-nx-tags.spec.ts

```diff
@@ -0,0 +1,39 @@
+let projects: { projectName: string; tags: string[] }[];
+
+const angularJson = require('../../angular.json');
+projects = Object.keys(angularJson.projects).map((projName) => {
+    return {
+        projectName: projName,
+        ...angularJson.projects[projName]
+    };
+});
+
+const knownTypes = new Set(['app', 'e2e', 'tools', 'lib']);
```

Comment:
I didn't do any separation for library types (like util, ui etc). Not sure it will be really useful, but let me know if you have any thoughts 🙂